proto/dymensionxyz/dymension/rollapp/rollapp.proto

```diff
@@ -17,31 +19,51 @@ message RollappGenesisState {
   bool transfers_enabled = 2;
 }
 
-// Rollapp defines a rollapp object. First the RollApp is created and then
+// Rollapp defines a rollapp object. First, the RollApp is created and then
 // sequencers can be created and attached. The RollApp is identified by rollappId
 message Rollapp {
   // The unique identifier of the rollapp chain.
-  // The rollappId follows the same standard as cosmos chain_id.
-  string rollappId = 1;
+  // The rollapp_id follows the same standard as cosmos chain_id.
+  string rollapp_id = 1;
   // creator is the bech32-encoded address of the rollapp creator.
   string creator = 2;
-  // version is the software and configuration version.
-  // starts from 1 and increases by one on every MsgUpdateState
-  uint64 version = 3;
-  // maxSequencers is the maximum number of sequencers.
-  uint64 maxSequencers = 4;
-  // permissionedAddresses is a bech32-encoded address list of the sequencers that are allowed to serve this rollappId.
-  // In the case of an empty list, the rollapp is considered permissionless.
-  repeated string permissionedAddresses = 5;
-  reserved 6;
+  // initial_sequencer_address is a bech32-encoded address of the
+  // sequencer that is allowed to initially serve this rollappId.
+  string initial_sequencer_address = 3;
+  // checksum used to verify integrity of the genesis file
+  string genesis_checksum = 4;
   // genesis_state is a partial repr of the state the hub can expect the rollapp to be in upon genesis
-  RollappGenesisState genesis_state = 7 [(gogoproto.nullable) = false];
+  RollappGenesisState genesis_state = 5 [(gogoproto.nullable) = false];
   // channel_id will be set to the canonical IBC channel of the rollapp.
-  string channel_id = 8;
+  string channel_id = 6;
   // frozen is a boolean that indicates if the rollapp is frozen.
-  bool frozen = 9;
-  // registeredDenoms is a list of registered denom bases on this rollapp
-  repeated string registeredDenoms = 10;
+  bool frozen = 7;
+  // unique bech32 prefix
+  string bech32_prefix = 8;
+  // alias is the chain alias used for display and namespace system
+  string alias = 9;
+  // registered_denoms is a list of registered denom bases on this rollapp
+  repeated string registered_denoms = 10;
+  // metadata is the rollapp metadata
+  RollappMetadata metadata = 11;
+}
```

Comment:
we migrate everything in the store anyway, so it's fine

Reply:
sorry how is it fine? you can't just say it's fine because you migrate because the migration will use the new type

Reply:
right I see, because you copy paste the old type to upgrade directory
you should keep the old proto in a upgrade proto directory I think

Reply:
also, does it work for historical queries?

Reply:
tbh I don't see why you can't just reserve the old fields, it's a lot simpler to think about and track lineage of stuff, and best practice
can you do it please

Reply:
come to think of it, considering the historical queries, now I'm not sure anymore. If you replace the fields instead of reserving them, and query the old data using the new types, it would likely fail to deserialize. 
My reasoning to replace the old fields was that if we keep removing fields, which we will likely do, we could end up with a proto with more `reserved` fields than actual ones